scripts/helpers/roll_docker_version.sh

```diff
@@ -38,31 +42,46 @@ for i in "$@"; do
     esac
 done
 
-if [[ ! $ROLL_REASON ]]; then
-    echo 'Roll reason required, please add --reason="Why I am rolling"'
+if [[ -z "$NO_DOCKER" ]]; then
+  if [[ ! $ROLL_REASON ]]; then
+    echo 'Roll reason required, please add --reason="Why I am rolling" or use --no-docker'
     exit -1
+  fi
+  if [[ -z "$NEXT_VERSION" ]]; then
+    NEXT_VERSION=$((CURRENT_VERSION + 1))
+  fi
+  echo "Current version: $CURRENT_VERSION"
+  echo "Next version: $NEXT_VERSION"
+  echo "Reason: $ROLL_REASON"
+else
+  if [[ -z "$NEXT_VERSION" ]]; then
+    NEXT_VERSION="$CURRENT_VERSION"
+  fi
+  echo "Updating to version: $NEXT_VERSION"
 fi
 
-echo "Current version: $CURRENT_VERSION"
-echo "Next version: $NEXT_VERSION"
-echo "Reason: $ROLL_REASON"
 echo ""
 
 echo "Rolling workflows"
 sed -r -i "s|image: ghcr\.io/project-chip/(.*):[0-9,a-z,A-Z,-]*(\..*)?|image: ghcr.io/project-chip/\1:$NEXT_VERSION|" .github/workflows/*.yaml
 sed -r -i "s|image: ghcr\.io/project-chip/(.*):[0-9,a-z,A-Z,-]*(\..*)?|image: ghcr.io/project-chip/\1:$NEXT_VERSION|" .github/workflows/*.yml
 
+echo "Rolling cloudbuild"
+sed -r -i "s|ghcr\.io/project-chip/(.*):[0-9,a-z,A-Z,-]*(\..*)?|ghcr.io/project-chip/\1:$NEXT_VERSION|" integrations/cloudbuild/*.yaml
+
 echo "Rolling VSCode container"
 sed -r -i "s|matter-dev-environment:local --version [0-9,a-z,A-Z,-]*|matter-dev-environment:local --version $NEXT_VERSION|" .devcontainer/devcontainer.json
 
-echo "Rolling docker images"
-find . -iname Dockerfile -not -path "./third_party/*" | xargs sed -r -i "s|ARG VERSION=[0-9,a-z,A-Z,-]*|ARG VERSION=$NEXT_VERSION|"
-
-echo "Rolling main docker version"
-echo "$NEXT_VERSION : $ROLL_REASON" >./integrations/docker/images/base/chip-build/version
-
 echo "Rolling README.md files"
 find . -iname readme.md -not -path "./third_party/*" | xargs sed -r -i "s|ghcr\.io/project-chip/(.*):[0-9,a-z,A-Z,-]*(\..*)?|ghcr.io/project-chip/\1:$NEXT_VERSION|"
 
+if [[ -z "$NO_DOCKER" ]]; then
```

Comment:
What is this for? This seems like it will just get us back to disparate versions

Reply:
I'll happily roll everything again to the current version if needed, we need to keep everything in sync.

Reply:
We already have versions of docker updated, no need to update them and this would cause a re-build and re-typing "reason" is not useful (the existing version already has a reason).

Reply:
In particular in this PR I just want to "update all workflows to latest version".

The script forces me to provide a "reason" and there is no such thing. It also forces me to manually type the version that is already there (and to not have changes i need to match the reason or revert) when it already knows what version I need.